tests/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import find_packages, setup
+
+setup(name="kafkatest",
+      version="0.8.3-SNAPSHOT",
```

Comment:
0.8.3-SNAPSHOT -> 0.8.3

Why? According to packaging standards (https://www.pypa.io/en/latest/peps/#pep426-metadata-for-python-software-packages-2-0), PEP440 should be used as a guide to versioning for pip and other packaging tools.